src/include/CMakeLists.txt

```diff
@@ -186,6 +186,9 @@ install (FILES kdbextension.h
 	       elektra.h
 	       kdbendian.h
 	       kdbmacros.h
+		   kdbmerge.h
+		   kdberrors.h
```

Comment:
This is a private file and should not be installed. @Chemin1 can you fix that kdbmerge.h does not include kdberrors.h nor kdblogger.h.

Reply:
Fixed in #3234